### DISCUSSION TITLE
why delete  "bin": {     "lerna": "core/lerna/cli.js"   }, in root package.json

### DISCUSSION THREAD

**Discussion by aidevtao** [lerna/lerna#3791] [Q&A]
why delete  "bin": {     "lerna": "core/lerna/cli.js"   }, in root package.json
I can find the main point of the lerna

**Comment by fahslaj** [comment]
Hi @aidevtao , I am not quite sure what you mean. The binary is specified in Lerna's package.json file. In a repo with Lerna installed, you can see it by looking in `node_modules/lerna/package.json`:
```
...
  "bin": {
    "lerna": "dist/cli.js"
  },
...
```
